docs/plugins.md

```diff
@@ -41,6 +41,28 @@ mvn clean compile
 
 For full details of all options, see the [plugin README](https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator-maven-plugin).
 
+### Dependencies
+
+The models generated use annotations like `@ApiModelProperty` the dependency commonly used is:
+
+```xml
+<dependency>
+    <groupId>io.swagger.core.v3</groupId>
+    <artifactId>swagger-annotations</artifactId>
+</dependency>
+```
+
+But this will not work. This depedency does not contains the package neither the classes needed. The resulting code it will fail to compile.
```

Comment:
```suggestion
But this will not work. This dependency is not binary compatible with Swagger v2 annotations. The resulting code will fail to compile.
```